tests/e2e/Adapter/Scopes/Relationships/ManyToOneTests.php

```
@@ -1677,4 +1677,61 @@
+        $database->createCollection('child');
+        $database->createAttribute('parent', 'name', Database::VAR_STRING, 255, true);
+        $database->createAttribute('child', 'name', Database::VAR_STRING, 255, true);
+
+        $database->createRelationship(
+            collection: 'child',
+            relatedCollection: 'parent',
+            type: Database::RELATION_MANY_TO_ONE,
+            onDelete: Database::RELATION_MUTATE_RESTRICT
+        );
+
+        $parent = $database->createDocument('parent', new Document([
```

PHPMD (2.15.0)

Avoid unused local variables such as '$parent'. (Unused Code Rules)

(UnusedLocalVariable)